graylog2-web-interface/public/stylesheets/graylog2.less

```diff
@@ -1312,6 +1312,21 @@ td.limited {
   padding-top: 1px;
 }
 
+.ldap-test-login-result {
```

Comment:
Maybe it makes sense to defines these classes in a separate style and load/bundle it using the style-loader? This way we would start to bundle styles together with the components that use them and we could decouple/decompose the big global less file.